packages/dev/s2-docs/pages/s2/styling.mdx

```diff
@@ -37,6 +37,14 @@ Colocating styles with your component code means:
 - Develop more efficiently – no switching files or writing selectors.
 - Refactor with confidence – changes are isolated; deleting a component removes its styles.
 
+<InlineAlert variant="informative">
+  <Heading>Important Note</Heading>
+  <Content>
+    Due to the atomic nature of the generated CSS rules, it is strongly recommended that you follow the best practices listed [below](#css-optimization).
+    Failure to do so can result in large number of duplicate rules and obtuse styling bugs.
+  </Content>
+</InlineAlert>
```

Comment:
Maybe overkill, but it felt like it was important to mention this up front so people realize that this is important

Reply:
hmm yeah it's a little strong. I think after we re-structure the getting started page to include this it'll be unnecessary here.